src/System.Private.CoreLib/src/System/Globalization/CultureData.Windows.cs

```diff
@@ -159,6 +160,39 @@ private unsafe bool InitCultureData()
             return true;
         }
 
+        // Wrappers around the GetLocaleInfoEx APIs which handle marshalling the returned
+        // data as either and Int or String.
+        internal static unsafe String GetLocaleInfoEx(String localeName, uint field)
+        {
+            // REVIEW: Determine the maximum size for the buffer
+            const int BUFFER_SIZE = 530;
+
+            char* pBuffer = stackalloc char[BUFFER_SIZE];
+            int resultCode = GetLocaleInfoEx(localeName, field, pBuffer, BUFFER_SIZE);
+            if (resultCode > 0)
+            {
+                return new String(pBuffer);
+            }
+
+            return null;
+        }
+
+        internal static unsafe int GetLocaleInfoExInt(String localeName, uint field)
+        {
+            const uint LOCALE_RETURN_NUMBER = 0x20000000;
+            field |= LOCALE_RETURN_NUMBER;
+            int value = 0;
+            GetLocaleInfoEx(localeName, field, (char*)&value, sizeof(int));
+            return value;
+        }
+
+        internal static unsafe int GetLocaleInfoEx(string lpLocaleName, uint lcType, void* lpLCData, int cchData)
+        {
+            // Debug.Assert(!GlobalizationMode.Invariant);
```

Comment:
Nit: Do we want to keep these commented out asserts? The pattern in the CoreRt seems to have been to ignore the existence of GlobalizationMode and InvariantMode even as comments.